editor/project_manager.cpp

```diff
@@ -1223,7 +1223,7 @@ ProjectList::Item ProjectList::load_project_data(const String &p_path, bool p_fa
 	PackedStringArray unsupported_features = ProjectSettings::get_unsupported_features(project_features);
 
 	uint64_t last_edited = 0;
-	if (FileAccess::exists(conf)) {
+	if (cf_err == OK) {
```

Comment:
Not really related. I just noticed that this checks for file existence unnecessarily (unless we care about invalid project cfgs).